src/RageSoundReader_ThreadedBuffer.cpp

```diff
@@ -220,26 +228,9 @@ void RageSoundReader_ThreadedBuffer::BufferingThread()
 			continue;
 		}
 
-		/* Sleep proportionately to the amount of data we buffered, so we
-		 * fill at a reasonable pace. */
-		float fTimeFilled = float(g_iReadBlockSizeFrames) / m_iSampleRate;
-		float fTimeToSleep = fTimeFilled / 2;
-		if( fTimeToSleep == 0 )
-			fTimeToSleep = float(g_iReadBlockSizeFrames) / m_iSampleRate;
-
-		if( m_Event.WaitTimeoutSupported() )
```

Comment:
Are we not supporting `WaitTimeoutSupported()` anymore?

Reply:
Sorry, I've put it back.